### DISCUSSION TITLE
ARCropolis and Skyline both installed but skin mods are not working!

### DISCUSSION THREAD

**Discussion by keys-smash** [Raytwo/ARCropolis#508] [Issues]
So after the 16.0.0 update for the Switch I was running into several crashes and decided to just start from scratch on my SD card. So I went ahead and reinstalled ARCropolis and Skyline to my SD card and go through the process of adding mods in the "ultimate" folder at the root of my SD card. While im at it I go ahead and install the Training Modpack as well, after booting up Smash on the title screen I can see the ARCropolis version and everything but none of the mods in the mods folder were loading. On the other hand the Training Modpack was working just fine so im at a loss of what the issue is.

**Comment by ZenmoDAr** [comment]
When launching the game, try holding the plus (+) button after the first click of the joycon in the loading screen to enable your mods

**Comment by Chavastral** [comment]
Did you find a solution? i have the exact same problem.

**Reply by ZenmoDAr** [reply]
Mods are installed but not activated, When launching the game, try holding the plus (+) button after the first click of the joycon in the loading screen to enable your mods
